chart/kubeapps/templates/apprepository-jobs-cleanup.yaml

```diff
@@ -22,12 +22,10 @@ spec:
       - name: kubectl
         image: {{ template "kubeapps.image" (list .Values.hooks.image .Values.global) }}
         command:
-        - kubectl
-        - delete
-        - apprepositories.kubeapps.com
-        - -n
-        - {{ .Release.Namespace }}
-        - --all
+        - /bin/sh
+        args:
+        - -c
+        - kubectl delete apprepositories.kubeapps.com -n {{ .Release.Namespace }} --all; kubectl delete secrets -n {{ .Release.Namespace }} -l app={{ template "kubeapps.name" $ }},release={{ $.Release.Name }}
```

Comment:
I think this would be more readable and extendable if we formatted it as a multi-line string like in https://github.com/helm/charts/blob/master/stable/consul/templates/consul-statefulset.yaml#L109-L113.